cnd/src/btsieve/bitcoin/transaction_pattern.rs

```diff
@@ -75,7 +76,7 @@ mod tests {
 
     fn create_outpoint(tx: &str, vout: u32) -> OutPoint {
         OutPoint {
-            txid: sha256d::Hash::from_hex(tx).unwrap(),
```

Comment:
Up until https://github.com/rust-bitcoin/rust-bitcoin/pull/349 is merged and we can upgrade the bitcoin library, I actually liked importing with the namespace to clearly identify this `Hash` as a transaction hash.